### PR TITLE
[HouseTimelineGramplet]Added[gramps42]

### PR DESCRIPTION
**Pull request by todef96**
Added HouseTimelineGramplet, Lists Residents of an Address by Timeline

### CODE REVIEW

**Comment by sam-m888** [comment]
Thanks for this addon.

A few observations:

When you add another "address" entry to any person  and view that the gramplet does not get updated, unless you exit gramps and restart gramps.

When embeded in dashboard, the information is clipped (does not wrap to window) unless you detach from the dashboard.  

Question is this gramplet meant to be used in a particular view & bottombar say for the Person categories bottombar?

The Help link currently goes to the usermanual instead of its own page eg:
https://gramps-project.org/wiki/index.php?title=Gramps_4.2_Wiki_Manual_-_Gramplets#House_Timeline

gramplet-menu
gramps42_plugins_housetimeline-example

**Comment by johnbickmore** [comment]
@todef96 Haven't tested or looked at code, can that text house be optional?  Or use a house [unicode symbol](https://en.wikipedia.org/wiki/Miscellaneous_Technical) ?

File: HouseTimelineGramplet/housetimeline.gpr.py [hunk 1]
```diff
@@ -12,4 +12,5 @@
          detached_height = 460,
          gramplet = 'HouseTimelineGramplet',
          gramplet_title=_("House Timeline"),
+         help_url='Gramps_4.2_Wiki_Manual_-_Gramplets#House_Timeline'
```
Comment (sam-m888) [inline HouseTimelineGramplet/housetimeline.gpr.py]: Please change to new url:
`help_url='House_Timeline_Gramplet'`

Thanks

**Comment by sam-m888** [comment]
>>  FIX: Gramplet now responds to editing a persons address and gets updated, Help link now goes to gramplets intended url

@todef96 I've moved your help text and image out of the Gramps user manual as your gramplet does not come with Gramps by default. The new help page for you is 

https://gramps-project.org/wiki/index.php/House_Timeline_Gramplet

Please update the link address 👍

**Comment by sam-m888** [comment]
Thank you for the update +1

**Comment by todef96** [comment]
@johnbickmore as requested, I decided to make a gramplet menu option to remove/change the house ascii.

**Comment by sam-m888** [comment]
Thank you

**Pull request merged**